src/Image.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Image = React.createClass({
+
+  propTypes: {
+    src: React.PropTypes.string,
```

Comment:
Do we need to explicit document these? Might be cleaner just to pass them along by spreading props.